localization/french/culture/traditions/wc_cultural_heritages_l_french.yml

```
@@ -1,53 +1,52 @@
 ﻿l_french:
- heritage_generic_label_desc:0 "Each [heritage|E] defines a common origin for related [cultures|E]' customs and mannerisms, either as something they descend from or have wholesale adopted."
- heritage_northern_name:0 "Northern"
- heritage_northern_collective_noun:0 "Northerns"
+ heritage_northern_name:0 "Nordique"
+ heritage_northern_collective_noun:0 "Nordiens"
  heritage_arathi_name:0 "Arathi"
  heritage_arathi_collective_noun:0 "Arathi"
- heritage_gilnean_name:0 "Gilnean"
- heritage_gilnean_collective_noun:0 "Gilneans"
- heritage_southern_name:0 "Southern"
- heritage_southern_collective_noun:0 "Southerners"
- heritage_orcish_name:0 "Orcish"
+ heritage_gilnean_name:0 "Gilnéen"
+ heritage_gilnean_collective_noun:0 "Gilnéens"
+ heritage_southern_name:0 "Sudique"
+ heritage_southern_collective_noun:0 "Sudiens"
+ heritage_orcish_name:0 "Orque"
```

```suggestion
 heritage_orcish_name:0 "Orc"
```
La traduction officielle garde "orc" au masculin même pour les adjectifs